docs/changelog/82409.yaml

```diff
@@ -0,0 +1,31 @@
+pr: 82409
```

Comment:
Can we link multiple PRs here? Ideally we would have one entry for these changes:

Allow doc-values only search on number types, date types, ...

This PR-centric view for release notes isn't great for highlighting larger efforts

Reply:
@ywelsch I agree that it's not ideal in this case. Our current release notes style has a one-line-per-PR structure, and the new process is more about automation and giving the input some kind of structure.

One simple option would be to only fill in the `highlight:` section for one of the PRs, and it would cover all the related work. The release highlights asciidoc doesn't typically reference PRs, so that ought to work. I pared back 2 of the YAML files, so if you were to write up the release highlight asciidoc in `docs/changelog/82409.yaml`, that'd be great.

If that doesn't work, then we'll need to devise a convention for input files that aren't attached to a specific PR. That might be useful anyway? 🤔